scipy/stats/_continuous_distns.py

```diff
@@ -6279,13 +6283,15 @@ class gennorm_gen(rv_continuous):
     -----
     The probability density function for `gennorm` is [1]_::
 
-                                     beta
-        gennorm.pdf(x, beta) =  ---------------  exp(-|x|**beta)
-                                2 gamma(1/beta)
+    .. math::
+
+        f(x, \beta) = \frac{\beta}{2 \Gamma(1/\beta)} \exp(-|x|^\beta)
```

Comment:
This line is responsible for the one test failure:

    scipy.stats.gennorm:30: (WARNING/2) Inline substitution_reference start-string without end-string.::

The problem seems to be `|x|` (I think, but not sure), which was fine before but not now that it's a math block. I don't quite get it though; there are similar math expressions elsewhere and the docstring is raw (`r"""`).

Anyone else spot the problem?

Reply:
My bad, sorry. I think I fixed it with https://github.com/scipy/scipy/pull/8865/commits/f44bcf67d49dd5260553b51614cde502eadbe174 It was `::` 4 lines above the line which produced the error.

Reply:
that was it, good catch!